regression/scripts/DC_10_noise/DC_10_noise.py

```diff
@@ -18,17 +18,17 @@ def main():
     noise = Correlations.shevell(weight_landing, number_of_engines, thrust_sea_level, thrust_landing)
     
     actual = Data()
-    actual.takeoff = 99.7
-    actual.side_line = 97.2
-    actual.landing = 105.2
+    actual.takeoff = 99.982372547196633
```

Comment:
Hmm, I think these values actually come from Shevell data as a test case. I think what you have done is right. However, we should save the original values in comments somewhere.